azure/scope/managedcontrolplane.go

```diff
@@ -738,18 +739,15 @@ func (s *ManagedControlPlaneScope) StoreClusterInfo(ctx context.Context, caData
 	clusterInfo := &corev1.ConfigMap{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      "cluster-info",
```

Comment:
nit: I think I also saw there was a constant for this next to the `"kubeconfig"` one.

Reply:
nice! I looked for it and couldn't see it before 👁️